.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+![TICKETS_AND_ISSUES]
+
+[[JIRA_BOARD_ABBREVIATION]-[JIRA_TASK_NUMBER]: [TITLE_OF_JIRA_TASK]](https://link-to-jira-board-task)
+
+![DESCRIPTION]
+
+Plain language explanation of what feature request, bug fix, or enhancement this pull request includes
+
+![CHECKLIST]
+- [ ] By checking this, I can confirm that I have done the following tasks before opening this pull request:
+
+- 📚 My Pull Request Title follows this pattern: 
+    ```[FEATURE_REQUEST|ENHANCEMENT|BUG_FIX|DOCS] [JIRA_BOARD_NUMBER]: [TITLE_OF_JIRA_TASK]```
```

Comment:
An example of a title here would be good to have, to make sure we get the formatting right (I'm a bit of a stickler and don't like small variations in formatting). Something like FEATURE_REQUEST MVP1-1: Change email address

Reply:
use the suggestion tool ^^ and I'll commit your change

Reply:
I hope this formatting works out properly! 

```suggestion
    ```[FEATURE_REQUEST|ENHANCEMENT|BUG_FIX|DOCS] [JIRA_BOARD_NUMBER]: [TITLE_OF_JIRA_TASK]```
    Example:
    ```FEATURE_REQUEST MVP1-1: Change email address```
```